index.html

```diff
@@ -2766,134 +2728,114 @@ <h3>Consumption</h3>
+  72                                   # text(18)
+     766572696669636174696f6e4d6574686f64 # "verificationMethod"
+  81                                   # array(1)
+     78 40                             # text(64)
+        6469643a6578616d706c653a313244334b6f6f574d4864727a6377706a6264725a7335474771455241766367715833623564707550745061396f743639796577 # "did:example:12D3KooWMHdrzcwpjbdrZs5GGqERAvcgqX3b5dpuPtPa9ot69yew"
           </pre>
         </section>
+      </section>
+    </section>
+    <section>
+      <h3>Converting Data between Core Representations</h3>
+      <p></p>
 
-        <section>
-          <h4>COSE signatures</h4>
+    <section>
+      <h3>CBOR to JSON</h3>
+      <p>Most of the types in CBOR have direct analogs in JSON.  The following constraints allow for lossless conversion from CBOR to JSON:  </p>
+      <ul>
+        <li>An integer (major type 0 or 1) becomes a JSON number.</li>
+        <li>An array (major type 4) becomes a JSON array.</li>
+        <li>A map (major type 5) becomes a JSON object. This is possible directly only if all keys are UTF-8 strings. A converter might also convert other keys into UTF-8 strings (such as by converting integers into strings containing their decimal representation); however, doing so introduces a danger of key collision. </li>
+        <li>False (major type 7, additional information 20) becomes a JSON false.</li>
+        <li>True (major type 7, additional information 21) becomes a JSON true.</li>
+        <li>Null (major type 7, additional information 22) becomes a JSON null.</li>
+        <li>A floating-point value (major type 7, additional information 25 through 27) becomes a JSON number if it is finite (that is, it can be represented in a JSON number); if the value is non-finite (NaN, or positive or negative Infinity), it is represented by the substitute value.</li>
+        <li>A byte string with an encoding hint (major type 6, tag number 21 through 23) is encoded as described and becomes a JSON string.</li>
+        <li>Indefinite-length items are made definite before conversion. </li>
+      </ul>
 
-          <p>
-A DID Document proof may be constructed using CBOR semantic tagging, such as
-tag 98 for CBOR Object Signing and Encryption (COSE) [[RFC8152]]
-          </p>
+      <p>
+        When interoperating with a text-based representation such as JSON or JSON-LD it is helpful to utilize additional sematics to aid convertion.  CBOR Tags numbers 21 to 23 indicate that a byte string value in DID document represented in CBOR might require a specific encoding to represent properly in JSON.
```

Comment:
Language like this should really be avoided. The point of the ADM and production and consumption processes is really that you should be able to losslessly convert between ANY of the representations. If you add custom instructions for the special CBOR->JSON case, then would you also need custom instructions for every single pair of representations e.g. CBOR->YAML or CBOR->JSON-LD or CBOR->XML, etc.?

I think the Production section in this PR is really strong (perhaps minus the mention of `@context` which I don't really understand), but this section here about the special CBOR->JSON conversion case should be removed, and the "Consumption" section should be filled instead to explain how to go from a CBOR DID document to the ADM.

Reply:
@peacekeeper I get where you are coming from.  My mandate was `lossless conversion` between JSON <--> CBOR and this give specific instructions.  I'm having a bit of difficulty with the Abstract Data Model as being too `abstract` for my taste.  If we were to focus on CDDL as representative of the Abstract Data Model, then this would be effortless.    CBOR <--> CDDL <--> JSON <--> CDDL <--> YAML.    Being new to spec text, I welcome help to nail this down and make it more concrete.

Reply:
I find this section helpful, especially given the lack of clarity regarding the ADM... I think its possible it could be refactored into the  "Data Model Section" and then referenced there, but thats probably not a good idea for this PR.

Reply:
While this information may be valuable, the mandate was:

> Clear specification in DID Core for producing and consuming CBOR such that it can be "losslessly converted" (via the abstract data model) to the other representations.

Any additional information or guidance for how to better ensure that the CBOR values may be converted into JSON might fit better in the implementation guide.

Reply:
We definitely should not be defining a direct conversion like this, or encouraging people to do this kind of thing. This is a semantically-driven specification and so the ADM is the only defined method of translation. Anything else (including treating JSON-LD as plain JSON) is liable to work based on luck.